json_schemas/briefs-digital-outcomes-and-specialists-digital-specialists.json

```diff
@@ -188,6 +194,7 @@
     "numberOfSuppliers",
     "organisation",
     "priceWeighting",
+    "requirementsLength",
```

Comment:
Does adding this new required field have any knock on effects for briefs which have already been published? If we need to edit them again our validation would fail.
Should we migrate already published briefs to include this field?